### PR TITLE
[DA-1950] Create AW2F cronjob

### PR DESCRIPTION
**Pull request by j-kanuch**
This PR creates the cron job for the AW2F manifest generation. The query to select feedback records was updated to use a threshold of 2/3 of the `genomic_manifest_file.record_count` of the `input_manifest_id`.

### CODE REVIEW

File: rdr_service/dao/genomics_dao.py [hunk 1]
```diff
@@ -1447,6 +1447,26 @@ def get_feedback_equals_record_count(self):
...
+                GenomicManifestFile.id == GenomicManifestFeedback.inputManifestFileId
+            ).filter(
+                GenomicManifestFeedback.ignoreFlag == 0,
+                GenomicManifestFeedback.feedbackComplete == 0,
+                GenomicManifestFeedback.feedbackRecordCount != 0,
+                GenomicManifestFeedback.feedbackRecordCount >= GenomicManifestFile.recordCount * theta,
+                GenomicManifestFeedback.feedbackManifestFileId.is_(None),
+            ).all()
+
+        return list(results)
```
Comment (dtharpeuno) [inline rdr_service/dao/genomics_dao.py]: Can probably remove the outer `list` declaration, `all()` returns a list.

File: rdr_service/genomic/genomic_job_controller.py [hunk 2]
```diff
@@ -63,6 +63,7 @@ def __init__(self, job_id,
         self.task_data = task_data
         self.bypass_record_count = False
         self.skip_updates = False
+        self.feedback_threshold = float(2/3)
```
Comment (dtharpeuno) [inline rdr_service/genomic/genomic_job_controller.py]: 👿 number!

**Review by dtharpeuno — APPROVED** [review]
Looks good, just one comment.

File: rdr_service/dao/genomics_dao.py (hunk 1 above)
Comment (j-kanuch) [reply rdr_service/dao/genomics_dao.py]: Thanks, I removed the redundant `list()`

**Review by dtharpeuno — APPROVED** [review]

**Pull request merged**